moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -185,7 +183,19 @@ const useColumns = ({
         valueFormatter: (value) =>
           !!value?.fund_name ? `${value?.fund_id} | ${value?.fund_name}` : "",
         renderEditCell: (props) => (
-          <FundAutocompleteComponent {...props} data={data.moped_funds} />
+          <LookupAutocompleteComponent
+            {...props}
+            name={"fund"}
+            options={data.moped_funds}
+            fullWidthPopper={true}
+            autocompleteProps={{
+              getOptionLabel: (option) =>
+                option.fund_id ? `${option.fund_id} | ${option.fund_name}` : "",
+              isOptionEqualToValue: (value, option) =>
+                value.fund_id === option.fund_id &&
+                value.fund_name === option.fund_name,
+            }}
+          />
```

Comment:
another spot where declaring an object out of the component scope would put my mind at ease about performance degradation 😇

Reply:
So you propose that we declare the entire `autocompleteProps` out of the scope of the useColumns hook, or the individual functions?